features/nationaliteit.feature

```diff
@@ -2,319 +2,278 @@
+      | reden beëindigen (64.10) | datum ingang geldigheid (85.10) |
+      | 410                      | 20190604                        |
+      En de persoon heeft nog een 'nationaliteit' met de volgende gegevens
+      | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) |
+      | 0001                  | 017                   | 20190602                        |
       Als personen wordt gezocht met de volgende parameters
-      | naam                | waarde                          |
-      | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 555550001                       |
-      | fields              | nationaliteiten                 |
+      | naam                | waarde                              |
+      | type                | RaadpleegMetBurgerservicenummer     |
+      | burgerservicenummer | 000000073                           |
+      | fields              | nationaliteiten                     |
       Dan heeft de response een persoon met een 'nationaliteit' met de volgende gegevens
-      | naam | waarde                  |
-      | type | BehandeldAlsNederlander |
+      | naam                              | waarde                                               |
+      | type                              | Nationaliteit                                        |
+      | nationaliteit.code                | 0001                                                 |
+      | nationaliteit.omschrijving        | Nederlandse                                          |
+      | redenOpname.code                  | 017                                                  |
+      | redenOpname.omschrijving          | Rijkswet op het Nederlanderschap 1984, art. 3, lid 1 |
+      | datumIngangGeldigheid.type        | Datum                                                |
+      | datumIngangGeldigheid.datum       | 2019-06-02                                           |
+      | datumIngangGeldigheid.langFormaat | 2 juni 2019                                          |
 
-    @proxy
-    Scenario: de persoon is vastgesteld niet-Nederlander
-      Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 999994748 |
-      En de persoon heeft een 'nationaliteit' met de volgende gegevens
-      | naam                               | waarde |
-      | bijzonder Nederlanderschap (65.10) | V      |
+    Scenario: nationaliteit is onjuist
+      Gegeven de persoon met burgerservicenummer '000000085' heeft een 'nationaliteit' met de volgende gegevens
+      | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) |
+      | 0100                  | 301                   | 20200713                        |
+      En de 'nationaliteit' is gecorrigeerd naar de volgende gegevens
+      | onjuist (84.10) | datum ingang geldigheid (85.10) |
```

Comment:
Hier hoeft onjuist niet zelf te worden gezet. Dit gebeurt in de automation code. Door het hier zelf te zetten wordt de verkeerde rij wordt onjuist gezet op de verkeerde rij.
Verder is het niet duidelijk wat er in deze scenario precies wordt gevalideerd/beschreven. Maak je een nationaliteit onjuist door een nieuwe rij met alleen een datum ingang geldigheid?

Reply:
Dit scenario verwijdered. Was in essentie dezelfde als "geactualiseerde ingangsdatum.